lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -264,6 +265,10 @@ class CategoryRenderer {
     const clumpTmpl = this.dom.cloneTemplate('#tmpl-lh-clump', this.templateContext);
     const clumpElement = this.dom.find('.lh-clump', clumpTmpl);
 
+    if (clumpId === 'warning') {
```

Comment:
do we want it to be an open `<details>` or do we want it to be like failed audits and use `renderUnexpandableClump`? It seems like it should maybe operate like `failed`, but I'm not sure how others feel on this point.

Reply:
renderUnexpandableClump's don't have titles/descriptions so .... let's stick with this approach.
(If it supported titles, I would have said we do that.)